.astylerc

```diff
@@ -4,10 +4,9 @@
 --indent-labels
 --indent-col1-comments
 --min-conditional-indent=0
---break-blocks
```

Comment:
Because several people (@Kijewski and @haukepetersen namely) already complained that I include so many empty lines of code (mostly generated by this a style option) ;).